src/Time/TimeSteppers/DormandPrince5.hpp

```diff
@@ -0,0 +1,260 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+/// \file
+/// Defines class DormandPrince5.
+
+#pragma once
+
+#include <cstddef>
+#include <cstdint>
+#include <ostream>
+#include <pup.h>
+
+#include "ErrorHandling/Assert.hpp"
+#include "ErrorHandling/Error.hpp"
+#include "Options/Options.hpp"
+#include "Parallel/CharmPupable.hpp"
+#include "Time/Time.hpp"
+#include "Time/TimeSteppers/TimeStepper.hpp"
+#include "Utilities/ConstantExpressions.hpp"
+#include "Utilities/Gsl.hpp"
+#include "Utilities/TMPL.hpp"
+
+/// \cond
+struct TimeStepId;
+namespace TimeSteppers {
+template <typename Vars, typename DerivVars>
+class History;
+}  // namespace TimeSteppers
+/// \endcond
+
+namespace TimeSteppers {
+
+/*!
+ * \ingroup TimeSteppersGroup
+ *
+ * The standard 5th-order Dormand-Prince time stepping method, given e.g. in
+ * Sec. 7.2 of \cite NumericalRecipes.
+ *
+ * \f{eqnarray}{
+ * \frac{du}{dt} & = & \mathcal{L}(t,u).
+ * \f}
+ * Given a solution \f$u(t^n)=u^n\f$, this stepper computes
+ * \f$u(t^{n+1})=u^{n+1}\f$ using the following equations:
+ *
+ * \f{eqnarray}{
+ * k^{(1)} & = & dt \mathcal{L}(t^n, u^n),\\
+ * k^{(i)} & = & dt \mathcal{L}(t^n + c_i dt,
+ *                              u^n + \sum_{j=1}^i-1 a_{ij} k^{(j)}),
```

Comment:
`{i-1}` (missing braces)

Reply:
Sorry for all the typos! Thanks for finding them @wthrowe